Meta/Lagom/Tools/CodeGenerators/LibWeb/GenerateWindowOrWorkerInterfaces.cpp

```diff
@@ -437,6 +438,13 @@ ErrorOr<int> ladybird_main(Main::Arguments arguments)
     VERIFY(!paths.is_empty());
     VERIFY(!base_paths.is_empty());
 
+#if !defined(AK_OS_WINDOWS)
+    // Increase the memory limit, because on OpenBSD the default limit is to low
+    // FIXME: Figure out why when building with sanitizers this requires ridiculous limits, as it fails even with 64GiB
```

Comment:
Also since this has only affected BSD systems so far, maybe let's limit this to FreeBSD+OpenBSD in scope?